ontologies/vocabs.ttl

```diff
@@ -168,6 +168,20 @@ gist:ThreatActorType
 	rdfs:subClassOf gist:StixCategoryObject ;
 	.
 
+gist:WindowsPEBinaryType
+	a owl:Class ;
+	rdfs:subClassOf gist:StixCategoryObject ;
+	skos:definition """STIX 2.1 description: 
+The Windows PE binary vocabulary is currently used in the following SCO(s):
+●      File (Windows PE Binary extension)
+
+An open vocabulary of Windows PE binary types."""^^xsd:string ;
+	skos:example ""^^xsd:string ;
+	skos:note ""^^xsd:string ;
+	skos:prefLabel "Windows™ PE Binary Vocabulary"^^xsd:string ;
+	gist:stixTerm "windows-pebinary-type-ov"^^xsd:string ;
+	.
```

Comment:
- Move to TBox
- Change IRI per gist convention to avoid camelcasing ambiguity with acronyms
- add examples 
- add note
- align prefLabel with IRI, remove trademark character - asking for trouble.
```suggestion
gist:WindowsPeBinaryType
	a owl:Class ;
	rdfs:subClassOf gist:StixCategoryObject ;
	skos:definition """STIX 2.1 description: 
The Windows PE binary vocabulary is currently used in the following SCO(s):
●      File (Windows PE Binary extension)
An open vocabulary of Windows PE binary types."""^^xsd:string ;
	skos:example "exe, dll"^^xsd:string ;
	skos:note "PE stands for Portable Executable"^^xsd:string ;
	skos:prefLabel "Windows PE Binary Type"^^xsd:string ;
	gist:stixTerm "windows-pebinary-type-ov"^^xsd:string ;
	.